e2e/compatibility_test.go

```diff
@@ -0,0 +1,195 @@
+package e2e
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"gotest.tools/assert"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"gotest.tools/fs"
+)
+
+const (
+	pollTimeout = 10 * time.Second
```

Comment:
Could we add like 30 seconds for timeout, I'm afraid we will get false negatives with so little timeouts